src/Symfony/Component/ErrorHandler/Tests/ErrorEnhancer/ClassNotFoundErrorEnhancerTest.php

```diff
@@ -27,6 +27,14 @@ public static function setUpBeforeClass(): void
                 continue;
             }
 
+            if ($function instanceof \Closure) {
+                print "*****\n";
+                var_dump($function);
+                print_r(spl_autoload_functions());
+                exit;
+                print "*****\n";
+            }
+
             // get class loaders wrapped by DebugClassLoader
             if ($function[0] instanceof DebugClassLoader) {
                 $function = $function[0]->getClassLoader();
```

Comment:
i think the offending line is here :) which returns any callable.

Reply:
@ro0NL 👏 good point.